public/js/controllers/status_controller.js

```diff
@@ -63,6 +63,8 @@ function doNotification () {
   newBlockNtfn.show()
 }
 
+let isRedicting = true
```

Comment:
`isRedirecting`? (spelling)

But since it starts as true, the variable is actually "not redirecting".  Perhaps you wanted to start with false, negate the test, and then set to true the first time it recognizes `'sync complete'`?